virt/virt-4-10-release-notes.adoc

```diff
@@ -113,7 +113,7 @@ Removed features are not supported in the current release.
 
 * The VM Import Operator has been removed from {VirtProductName} with this release. It is replaced by the link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2[Migration Toolkit for Virtualization].
 
-* This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021.
+* This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021. However, CentOS Stream 8 can be used with upstream support from the user community.
```

Comment:
```suggestion
* This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021. However, CentOS Stream 8 can be used with community support.
```
I do not expect the reader to understand the term "upstream".

Reply:
Got it, thanks @dominikholler